_data/file_formats/eff_v1/header.yml

```diff
@@ -40,7 +40,6 @@
   type: dword
 - desc: Saving throw bonus
   type: dword
-- desc: Unknown
+- desc: Special (EE only...?)
```

Comment:
Not ee-only. And the next one should be save-for-half-damage.

Reply:
> Not ee-only. And the next one should be save-for-half-damage.

What do you mean by _next one_?

Reply:
next word

Reply:
> next word

There's no "next word", V1 Effects are `0x30` bytes long, the `Special` field is the last field (for both [SPL](https://gibberlings3.github.io/iesdp/file_formats/ie_formats/spl_v1.htm#splv1_Feature_Block) and [ITM](https://gibberlings3.github.io/iesdp/file_formats/ie_formats/itm_v1.htm#itmv1_Feature_Block) files).

Guess I could replicate what's there about `TobEx: Stacking ID` for non-EE games...

Reply:
According to our code, Special is not a dword, that's why. I haven't done any testing though.

Reply:
> According to our code, Special is not a dword, that's why. I haven't done any testing though.

If that's the case, I'll keep things as they are.
I think it's better to add links to [SPL](https://gibberlings3.github.io/iesdp/file_formats/ie_formats/spl_v1.htm#splv1_Feature_Block) and [ITM](https://gibberlings3.github.io/iesdp/file_formats/ie_formats/itm_v1.htm#itmv1_Feature_Block) files...